ports/stm32/boards/Passport/modules/tasks/copy_psbt_file_to_external_flash_task.py

```diff
@@ -34,6 +34,10 @@ async def copy_psbt_file_to_external_flash_task(on_done, on_progress, filename,
             psbt_len = fd.seek(0, 2)
             fd.seek(0)
 
+            if psbt_len > MAX_TXN_LEN:
+                await on_done(0, None, Error.PSBT_INVALID)
```

Comment:
We should differentiate between invalid PSBTs and ones that are just too long for us.  Add another error like `Error.PSBT_TOO_LARGE` and then show a different error in the code that calls this task.